All_home_works/hw2/task1.py

```diff
@@ -0,0 +1,96 @@
+"""
+Given a file containing text. Complete using only default collections:
```

Comment:
> Complete using only default collections:

What does it mean?

Reply:
I completely don't understand what does it mean. Because Counter and Deque are being imported from collections, it's a built-in library. So we can understand this phrase ("Complete using only default collections:") both: Do not use it at all and use only it. Nonsense.

Reply:
You need to take a lecture into the account, is about builtin collections. In that case, default is means builtin.
The idea of this task is to practice low-level operation.